modules/nw-ingress-setting-thread-count.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-setting-thread-count_{context}"]
+= Setting Ingress Controller thread count
+A cluster administrator can set the thread count to increase the amount of incoming connections a cluster can handle. You can patch an existing Ingress Controller to increase the amount of threads.
+
+. Procedure
+The following assumes that you already created an Ingress Controller.
+
+.. Update the Ingress Controller to increase the number of threads
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge -p '{"spec":{"tuningOptions": {"threadCount": 8}}}'
+----
+
+[IMPORTANT]
```

Comment:
If the user does not do this, will it cause a problem for their cluster, or is it more of an optimization? If the latter, consider using a `[TIP]` or `[NOTE]` admonition here.